include/mysql.h

```diff
@@ -216,6 +216,7 @@ extern const char *SQLSTATE_UNKNOWN;
     MYSQL_OPT_MAX_ALLOWED_PACKET,
     MYSQL_OPT_NET_BUFFER_LENGTH,
     MYSQL_OPT_TLS_VERSION,
+    MYSQL_OPT_USE_REDIRECTION,
```

Comment:
If this option is not available in MySQL Connector/C, it should be prefixed with MARIADB instead of MYSQL.